deb/openmediavault-lvm2/usr/share/openmediavault/engined/rpc/logicalvolumemgmt.inc

```diff
@@ -614,12 +617,74 @@ class OMVRpcServiceLogicalVolumeMgmt extends \OMV\Rpc\ServiceAbstract {
 				"vgname" => $lv->getVGName(),
 				"attributes" => $lv->getAttributes(),
 				"vgattributes" => $lv->getVGAttributes(),
-				"isavailable" => $lv->IsMediaAvailable()
+				"isavailable" => $lv->IsMediaAvailable(),
+				"_used" => $used
 			];
 		}
 		return $result;
 	}
 
+	/**
+	 * Checks /proc/mount for a mount point for a given LVM logical volume path.
+	 *
+	 * @param string $lvPath The full path of the LVM logical volume (can be /dev/dm-0, /dev/vgname/lvname, or /dev/mapper/vgname-lvname).
+	 * @return boolean true if a mount point is found
+	 */
+	private function checkForMountedFilesystem($lvPath) {
+		// Read the /proc/mounts file
+		$mounts = file('/proc/mounts', FILE_IGNORE_NEW_LINES | FILE_SKIP_EMPTY_LINES);
+		// Get all paths for the logical volume
+		$paths = $this->getLvmPaths($lvPath);
+		// Iterate through each line of /proc/mounts
+		foreach ($mounts as $line) {
+			$columns = preg_split('/\s+/', $line);
+			if (count($columns) < 1) {
+				continue;
+			}
+			$device = $columns[0];
+			// Check if the device is in the array of volumes
+			if (in_array($device, $paths, true)) {
+				return true;
+			}
+		}
+		return false;
+	}
+
+        /**
+         * Get paths for logical volume
+         * @param string $path any logical volume path
+         * @return An array containing three logical volume paths.
+         */
+	private function getLvmPaths($path) {
```

Comment:
I think this function can be replaced by the `getDeviceFileSymlinks` function. You can call it with `$lv->getDeviceFileSymlinks()`.

On my test system udevadm returns this:
```
DEVLINKS=/dev/vg0/lv0 /dev/disk/by-id/dm-uuid-LVM-x3YAcIjrdCGAMaFA6lfp1bEh2T3uKieqsBYoFUyzoJCp1c48CypereE9M2F9ofHO /dev/mapper/vg0-lv0 /dev/disk/by-id/dm-name-vg0-lv0
```